projects/kit/components/slider/helpers/slider-key-steps.directive.ts

```diff
@@ -1,24 +1,59 @@
-import {Directive, forwardRef, inject, Input} from '@angular/core';
+import {
+    Directive,
+    forwardRef,
+    // eslint-disable-next-line no-restricted-syntax
+    HostBinding,
+    inject,
+    Input,
+    signal,
+} from '@angular/core';
+import type {TuiValueTransformer} from '@taiga-ui/cdk/classes';
 import {TuiControl} from '@taiga-ui/cdk/classes';
 import {tuiFallbackValueProvider} from '@taiga-ui/cdk/tokens';
-import {tuiClamp} from '@taiga-ui/cdk/utils/math';
 
 import {TuiSliderComponent} from '../slider.component';
 import type {TuiKeySteps} from './key-steps';
-import {tuiKeyStepValueToPercentage, tuiPercentageToKeyStepValue} from './key-steps';
+import {tuiCreateKeyStepsTransformer} from './key-steps';
 
 @Directive({
     standalone: true,
     selector: 'input[tuiSlider][keySteps]',
+})
+export class TuiSliderKeyStepsBase {
```

Comment:
Let's rename it to `TuiSliderKeyStepsDirective` so people will know not to import it separately